src/styles/card.css

```diff
@@ -0,0 +1,105 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+}
+
+.card_img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+}
+
+.card_description {
+  color: #060b35;
```

Comment:
you defined this color as a variable but you did not use it here

Reply:
I have defined that color in scss so in css it aint gonna be a variable I think, am I right?